generator/config/expression/case.yaml

```diff
@@ -0,0 +1,21 @@
+# $schema: ../schema.json
+name: $case
+link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/switch/'
```

Comment:
I see the interest of creating this "fake" operator for code generation. It should be consistent with [`OutputWindow`](https://github.com/mongodb/mongo-php-builder/blob/592b0a0a1c60b3e412c1cfa91da8262c7a83fd72/src/Builder/Accumulator.php#L30-L37) this is the same need.

Could you create `accumulator/outputWindow.yaml` also?

Reply:
I'll do so in a separate PR: the current `OutputWindow` type contains additional validation logic that we can't represent through the config.